help/destinations/catalog/advertising/pubmatic.md

```diff
@@ -0,0 +1,129 @@
+
+### Authenticate to destination {#authenticate}
+
+To authenticate to the destination, fill in the required fields and select **[!UICONTROL Connect to destination]**.
+
+![How to authenticate](/help/destinations/assets/catalog/advertising/pubmatic/authenticate-destination.png)
+
+- **[!UICONTROL Bearer token]**: Fill in the bearer token to authenticate to the destination.
+
+### Fill in destination details {#destination-details}
+
+To configure details for the destination, fill in the required and optional fields below. An asterisk next to a field in the UI indicates that the field is required.
+
+![Destination details](/help/destinations/assets/catalog/advertising/pubmatic/destination-details.png)
+
+- **[!UICONTROL Name]**: A name by which you will recognize this destination in the future.
+- **[!UICONTROL Description]**: A description that will help you identify this destination in the future.
+- Data partner ID: The data partner ID set up in you PubMatic account for this integration.
+- Default Country Code: The default country code that should be applied to all identities if none is provided in the profile.
+- **[!UICONTROL Account ID]**: Your PubMatic Connect account ID.
+- Account type: The account type of your PubMatic platform account. These are the available options:
+  - PUBLISHER
+  - Demand Partner
+  - BUYER
+
+### Enable alerts {#enable-alerts}
+
+You can enable alerts to receive notifications on the status of the dataflow to your destination. Select an alert from the list to subscribe to receive notifications on the status of your dataflow. For more information on alerts, see the guide on [subscribing to destinations alerts using the UI](../../ui/alerts.md).
+
+When you are finished providing details for your destination connection, select **[!UICONTROL Next]**.
+
+## Activate segments to this destination {#activate}
+
+> [!IMPORTANT]
+>
+> - To activate data, you need the **[!UICONTROL Manage Destinations]**, **[!UICONTROL Activate Destinations]**, **[!UICONTROL View Profiles]**, and **[!UICONTROL View Segments]** [access control permissions](/help/access-control/home.md#permissions). Read the [access control overview](/help/access-control/ui/overview.md) or contact your product administrator to obtain the required permissions.
+>
+> * To export _identities_, you need the **[!UICONTROL View Identity Graph]** [access control permission](/help/access-control/home.md#permissions). <br> ![Select identity namespace highlighted in the workflow to activate audiences to destinations.](/help/destinations/assets/overview/export-identities-to-destination.png "Select identity namespace highlighted in the workflow to activate audiences to destinations."){width="100" zoomable="yes"}
+
+Read [Activate profiles and segments to streaming segment export destinations](/help/destinations/ui/activate-segment-streaming-destinations.md) for instructions on activating audience segments to this destination.
+
+### Map attributes and identities {#map}
```

Comment:
This section would really benefit from some screenshots to exemplify a correct mapping.